server/api/assessments.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const router = express.Router();
+const assessmentDb = require("../services/database/assessments");
+
+router.get("/", (req, res) => {
+	assessmentDb
+		.getAllAssessments()
+		.then((data) => {
+			res.send(data);
```

Comment:
In here is better returning the data in JSON format. You can easily do that by using `res.json()` instead of `res.send()`.

Reply:
Done